### ISSUE TITLE
when i train resnet the program stuck

### ISSUE THREAD

**Issue by andongchen** [tensorflow/models#1036]
python3.5 tf version 1.0 the program is stuck here
=========================Options=============================
-max_depth                  10000
-min_bytes                  0
-min_micros                 0
-min_params                 0
-min_float_ops              1
-device_regexes             .*
-order_by                   float_ops
-account_type_regexes       .*
-start_name_regexes         .*
-trim_name_regexes          
-show_name_regexes          .*
-hide_name_regexes          
-account_displayed_op_only  true
-select                     float_ops
-viz                        false
-dump_to_file               

==================Model Analysis Report======================
_TFProfRoot (0/17.63b flops)
  unit_2_1/sub2/conv2/Conv2D (603.98m/603.98m flops)
  unit_3_4/sub2/conv2/Conv2D (603.98m/603.98m flops)
  unit_3_4/sub1/conv1/Conv2D (603.98m/603.98m flops)
  unit_3_3/sub2/conv2/Conv2D (603.98m/603.98m flops)
  unit_3_3/sub1/conv1/Conv2D (603.98m/603.98m flops)
  unit_3_2/sub2/conv2/Conv2D (603.98m/603.98m flops)
  unit_3_2/sub1/conv1/Conv2D (603.98m/603.98m flops)
  unit_3_1/sub2/conv2/Conv2D (603.98m/603.98m flops)
  unit_3_1/sub1/conv1/Conv2D (603.98m/603.98m flops)
  unit_3_0/sub2/conv2/Conv2D (603.98m/603.98m flops)
  unit_2_4/sub2/conv2/Conv2D (603.98m/603.98m flops)
  unit_2_4/sub1/conv1/Conv2D (603.98m/603.98m flops)
  unit_2_3/sub2/conv2/Conv2D (603.98m/603.98m flops)
  unit_2_3/sub1/conv1/Conv2D (603.98m/603.98m flops)
  unit_2_2/sub2/conv2/Conv2D (603.98m/603.98m flops)
  unit_2_2/sub1/conv1/Conv2D (603.98m/603.98m flops)
  unit_2_1/sub1/conv1/Conv2D (603.98m/603.98m flops)
  unit_2_0/sub2/conv2/Conv2D (603.98m/603.98m flops)
  unit_1_0/sub1/conv1/Conv2D (603.98m/603.98m flops)
  unit_1_4/sub2/conv2/Conv2D (603.98m/603.98m flops)
  unit_1_4/sub1/conv1/Conv2D (603.98m/603.98m flops)
  unit_1_3/sub2/conv2/Conv2D (603.98m/603.98m flops)
  unit_1_3/sub1/conv1/Conv2D (603.98m/603.98m flops)
  unit_1_0/sub2/conv2/Conv2D (603.98m/603.98m flops)
  unit_1_2/sub2/conv2/Conv2D (603.98m/603.98m flops)
  unit_1_2/sub1/conv1/Conv2D (603.98m/603.98m flops)
  unit_1_1/sub2/conv2/Conv2D (603.98m/603.98m flops)
  unit_1_1/sub1/conv1/Conv2D (603.98m/603.98m flops)
  unit_3_0/sub1/conv1/Conv2D (301.99m/301.99m flops)
  unit_2_0/sub1/conv1/Conv2D (301.99m/301.99m flops)
  init/init_conv/Conv2D (113.25m/113.25m flops)
  logit/xw_plus_b (1.28k/165.12k flops)
    logit/xw_plus_b/MatMul (163.84k/163.84k flops)
  gradients/logit/xw_plus_b/MatMul_grad/MatMul_1 (163.84k/163.84k flops)
  gradients/logit/xw_plus_b/MatMul_grad/MatMul (163.84k/163.84k flops)

======================End of Report==========================
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE3 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:910] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
I tensorflow/core/common_runtime/gpu/gpu_device.cc:885] Found device 0 with properties: 
name: GeForce GTX 1060 6GB
major: 6 minor: 1 memoryClockRate (GHz) 1.7845
pciBusID 0000:01:00.0
Total memory: 5.93GiB
Free memory: 5.12GiB
I tensorflow/core/common_runtime/gpu/gpu_device.cc:906] DMA: 0 
I tensorflow/core/common_runtime/gpu/gpu_device.cc:916] 0:   Y 
I tensorflow/core/common_runtime/gpu/gpu_device.cc:975] Creating TensorFlow device (/gpu:0) -> (device: 0, name: GeForce GTX 1060 6GB, pci bus id: 0000:01:00.0)

**Comment by aselle** [comment]
@panyx0718 , could you take a look please! Thanks!

**Comment by andongchen** [comment]
I still don't known how to do!
could you explain in detail pelase,thanks!

**Comment by panyx0718** [comment]
Hi, it didn't stuck. It was training without printing out anything. You can use tensorboard to see the results. Or you can add some printf to the code if you want.

**Comment by prb12** [comment]
Closing due to lack of activity.
